packages/darma-vt/package.py

```diff
@@ -55,7 +56,7 @@ class DarmaVt(CMakePackage):
     homepage = "https://github.com/DARMA-tasking/vt"
     git = "https://github.com/DARMA-tasking/vt.git"
 
-    version("1.0.0", tag="1.0.0")
+    version("1.5.0", tag="1.5.0")
```

Comment:
I think we should just add version 1.5 (ie. specify both versions are available)

Reply:
Understood, so I am approving it @pierrepebay you can now merge, tx